libpkpass/commands/create.py

```diff
@@ -1,5 +1,5 @@
 """This module allows for the creation of passwords"""
-import getpass
+import getpass, sys
```

Comment:
We prefer imports on separate lines https://legacy.python.org/dev/peps/pep-0008/#imports

Reply:
Sounds good